examples/layouts/sidebar_app.R

```diff
@@ -0,0 +1,80 @@
+library(shiny)
+library(shiny.semantic)
+library(glue)
```

Comment:
Glue is not used in this example, isn't it?

Reply:
Exactly, I removed that